<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-7HNKN0S" name="com.itc.calc.testcases.CalorieCalculatorDDT_ExcelTestCase" tests="1" failures="0" timestamp="9 Apr 2021 09:00:28 GMT" time="0.000" errors="1">
  <testcase name="EnterCalorieDetails" time="0.000" classname="com.itc.calc.testcases.CalorieCalculatorDDT_ExcelTestCase">
    <error type="java.lang.RuntimeException" message="java.io.FileNotFoundException: F:\Trainings_Apr2021\TL_ITC_SeleniumJava_Apr2021\Selenium_Workspace\MavenProject\src\com\itc\calc\testdata\CalorieTestDataSet.xlsx (The system cannot find the path specified)">
      <![CDATA[java.lang.RuntimeException: java.io.FileNotFoundException: F:\Trainings_Apr2021\TL_ITC_SeleniumJava_Apr2021\Selenium_Workspace\MavenProject\src\com\itc\calc\testdata\CalorieTestDataSet.xlsx (The system cannot find the path specified)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:162)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:441)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1303)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1019)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1108)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1200)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1125)
	at org.testng.TestNG.run(TestNG.java:1033)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
Caused by: java.io.FileNotFoundException: F:\Trainings_Apr2021\TL_ITC_SeleniumJava_Apr2021\Selenium_Workspace\MavenProject\src\com\itc\calc\testdata\CalorieTestDataSet.xlsx (The system cannot find the path specified)
	at java.io.FileInputStream.open0(Native Method)
	at java.io.FileInputStream.open(FileInputStream.java:195)
	at java.io.FileInputStream.<init>(FileInputStream.java:138)
	at com.itc.calc.utilities.ExcelReader.ReadFromExcelToObjArr(ExcelReader.java:15)
	at com.itc.calc.testcases.CalorieCalculatorDDT_ExcelTestCase.getCalorieData(CalorieCalculatorDDT_ExcelTestCase.java:82)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:136)
	... 20 more
]]>
    </error>
  </testcase> <!-- EnterCalorieDetails -->
</testsuite> <!-- com.itc.calc.testcases.CalorieCalculatorDDT_ExcelTestCase -->
